x/staking/keeper/genesis.go

```diff
@@ -3,6 +3,7 @@ package keeper
 import (
 	"context"
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/types/module"
```

Comment:
add this import in L14 instead to fix lint

Reply:
done!

Reply:
done!